src/Summoner/Template/Cabal.hs

```diff
@@ -79,8 +79,8 @@ cabalFile Settings{..} = File (toString settingsRepo ++ ".cabal") cabalFileConte
           exposed-modules:     $libModuleName
                                $preludeMod
 
-          build-depends:       $settingsBase
-                             $commaPreludeLibrary
+          build-depends:         $settingsBaseType
+                               $commaPreludeLibrary
```

Comment:
```suggestion
                             $commaPreludeLibrary
```

Reply:
It is now fixed. I rebased to keep the commit history clean.